CMakeLists.txt

```diff
@@ -445,18 +449,44 @@ elseif(UNIX)
   list(APPEND mxnet_LINKER_LIBS Threads::Threads)
 endif()
 
+if(USE_ILP64_LAPACKE)
+  message("USE_ILP64_LAPACKE is ON")
+  if(NOT USE_INT64_TENSOR_SIZE)
+    message(FATAL_ERROR "USE_INT64_TENSOR_SIZE must be set to ON when USE_ILP64_LAPACKE=ON")
+  endif()
+  if(NOT USE_LAPACKE_INTERFACE)
+    message("Automatically set USE_LAPACKE_INTERFACE=ON")
+    set(USE_LAPACKE_INTERFACE ON)
+  endif()
+  add_definitions(-DMXNET_USE_ILP64_LAPACKE=1)
+endif()
+
+if(USE_LAPACKE_INTERFACE)
+  message("USE_LAPACKE_INTERFACE is ON")
+  if(NOT USE_LAPACK)
+    message("Automatically set USE_LAPACK=ON")
+    set(USE_LAPACK ON)
+  endif()
+  add_definitions(-DMXNET_USE_LAPACKE_INTERFACE=1)
+endif()
```

Comment:
Because users might still want to do apt-get openblas and apt-get lapack if they do not want to build openblas from source. Also edge, mac os and windwos pipelines still use LAPACKE=0